### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 25

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#201] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152254248-fa932810-a5db-4b1b-a72a-cba6b61c5761.png)

**Comment by Exyss** [comment]
Il problema si traduce nella seguente forma: 
- 12 pallini
- 3-1 separatori

1. $\binom{12+3-1}{3} = \binom{14}{2}$
2. Consideriamo 6 pallini come già distribuiti: $\binom{12-6+3-1}{3-1} = \binom{8}{2}$
3. Soluzione analoga a #26: $\binom{14}{2} - 3 \binom{10}{2} + 3 \binom{6}{2} - \binom{2}{2}$
